src/backend/utilities/sessionStorage.ts

```diff
@@ -13,8 +13,8 @@ import { sessionHeader } from '../endpoints/sessionHeader';
 
 export interface BasicSession {
   sessionId: string;
-  did?: IDidDetails['did'];
-  encryptionKeyId?: DidPublicKey['id'];
+  did?: IDidDetails['uri'];
+  encryptionKeyUri?: DidPublicKey['uri'];
```

Comment:
Should we use the new types now?
```suggestion
  did?: DidUri;
  encryptionKeyUri?: DidResourceUri;
```